ui/src/app/modules/login/login.component.ts

```diff
@@ -10,6 +10,7 @@ export class LoginComponent {
   private loginError :string
 
   constructor(private router: Router, private route: ActivatedRoute) {
+    localStorage.removeItem('id_token');
```

Comment:
Yea, my thought was, we check everytime the app is visited if there is a token and if it's valid (I think), and within that method if it errors out for any reason we should remove the token (currently I believe it just redirects and the expired or invalid token remains in localStorage)